.github/README.md

```diff
@@ -688,6 +688,7 @@ Here is a list of middleware that are included within the Fiber framework.
 | [idempotency](https://github.com/gofiber/fiber/tree/main/middleware/idempotency)     | Allows for fault-tolerant APIs where duplicate requests do not erroneously cause the same action performed multiple times on the server-side.                           |
 | [keyauth](https://github.com/gofiber/fiber/tree/main/middleware/keyauth)             | Adds support for key based authentication.                                                                                                                              |
 | [limiter](https://github.com/gofiber/fiber/tree/main/middleware/limiter)             | Adds Rate-limiting support to Fiber. Use to limit repeated requests to public APIs and/or endpoints such as password reset.                                             |
+| [loadshedding](https://github.com/gofiber/fiber/tree/main/middleware/loadshedding)   | Gracefully manages server load by enforcing request timeouts and handling resource-intensive requests during high-traffic periods.                                      |
```

Comment:
what is the difference to the timeout middleware? could you also store a handler in the config that handles these cases?